index.bs

```diff
@@ -812,18 +851,24 @@ spec:css-syntax-3;
         5.  Let |choice| be the result of <a abstract-op lt="ask to choose">asking the user to
             choose a `Credential`</a>, given |options| and |credentials|.
 
-        6.  If |choice| is `null` or a {{Credential}}, [=resolve=] |p| with |choice| and skip the
-            remaining steps.
+        6.  If |choice| is `null` or a {{Credential}}, let |result| be |choice| and go to step 9.
```

Comment:
Instead of "if ... goto", I'd use "if not ... then {...}"

Reply:
Done. I don't know why I'm so tempted to use goto on specs when I've only ever used it once? in code.